invenio_rdm_records/resources/serializers/__init__.py

```diff
@@ -42,4 +43,6 @@
     "StringCitationSerializer",
     "UIJSONSerializer",
     "DCATSerializer",
+    "CFFSerializer",
```

Comment:
minor: I think this is not imported anywhere above.

Reply:
Leftover from the CFF serializer implementation, thanks!